app/src/serial-cmdhandler.cpp

```diff
@@ -1,26 +1,25 @@
 #include "serial-cmdhandler.h"
 #include "app-settings.h"
 #include "utils.h"
-
 //------------------------------------------------------------------------------
 #define SERIAL_CMD_HANDLER_DELIM ","
 #define SERIAL_CMD_HANDLER_TERM1 '\n' // LF
 #define SERIAL_CMD_HANDLER_TERM2 '\r' // CR
-
-
+#define SERIAL_CMD_HANDLER_TERM3 '[A' // arrowup
+#define SERIAL_CMD_HANDLER_TERM4 '\b' // backspace
+#define SERIAL_CMD_HANDLER_TERM5 '[B' // arrowdown
 //------------------------------------------------------------------------------
-SerialCmdHandler::SerialCmdHandler(Commander& cmdr)
-    : CommandHandler(SERIAL_CMD_HANDLER_DELIM, SERIAL_CMD_HANDLER_TERM1, SERIAL_CMD_HANDLER_TERM2)
-    , m_cmdr(cmdr)
+SerialCmdHandler::SerialCmdHandler(Commander &cmdr)
+    : CommandHandler(SERIAL_CMD_HANDLER_DELIM, SERIAL_CMD_HANDLER_TERM1, SERIAL_CMD_HANDLER_TERM2, SERIAL_CMD_HANDLER_TERM3, SERIAL_CMD_HANDLER_TERM4, SERIAL_CMD_HANDLER_TERM5), m_cmdr(cmdr)
 {
     // Setup callbacks for SerialCommand commands
     setCommands();
-    for (auto& cmd : m_commands)
+    for (auto &cmd : m_commands)
     {
         addCommand(cmd.first.c_str(), cmd.second);
     }
-
     setDefaultHandler(std::bind(&SerialCmdHandler::processCmdUnrecognized, this));
+    addCommand("st", std::bind(&SerialCmdHandler::processCmdRelayGetSate, this)); // adds new command st for relay state check

```

Comment:
wrong place, move it to SerialCmdMap{} in SerialCmdHandelr::setCommands()

Reply:
done